Makefile

```diff
@@ -953,6 +956,15 @@ else
 	@echo Cannot run an astyle check, your system either does not have astyle, or it is too old.
 endif
 
+style-json: new_json_whitelist json_formatter
+	cat new_json_whitelist | xargs -L 1 ./json_formatter
+
+style-all-json: json_formatter
+	find data -name "*.json" | xargs -L 1 ./json_formatter
```

Comment:
Meh, I almost want to make paths with whitespaces break because why the hell would you do that, but sure.